### ISSUE TITLE
Glass Panes rendering middle seam

### ISSUE THREAD

**Issue by Endoman123** [Chisel-Team/ConnectedTexturesMod#124]
Hi, as the title says, glass panes are rendering the middle seam even when connected.

I am using OptiFine with Connected Textures disabled.

![image](https://user-images.githubusercontent.com/7063037/84733182-87a01c00-af6b-11ea-80d7-cf3a56cd69dd.png)

**Comment by ChloeDawn** [comment]
Does this occur without OptiFine?

**Comment by Endoman123** [comment]
> Does this occur without OptiFine?

It does. With more testing, it seems to be a problem with the glass model definition.
I have not been able to fix it, and using the model JSON from the ConnectedTexturesPack doesn't fix it. Instead there's an issue very similar to Chisel-Team/ConnectedTexturesMod#107.

![image](https://user-images.githubusercontent.com/7063037/85215110-df5ece80-b341-11ea-9347-03a60b6bb927.png)

**Comment by ChloeDawn** [comment]
Yeah this is related to block occlusion code. It should be occluding vertically for matching panes. Vanilla skips this occlusion here iirc. I'm not sure the best way for CTP/CTM to work around this.

**Comment by tterrag1098** [comment]
We have a working model in Chisel that should be ported over to this.
